docker-compose.yml

```diff
@@ -353,7 +353,7 @@ services:
             - MAX_BRIDGE_PARTICIPANTS
             - OCTO_BRIDGE_SELECTION_STRATEGY
             - PROSODY_VISITORS_MUC_PREFIX
-            - SENTRY_DSN="${JICOFO_SENTRY_DSN:-0}"
```

Comment:
This removes support for setting individual DSN values.

SENTRY_DSN is the name the environment variable the sentry library enforces to use.

Reply:
The variable is read during templating, please see the logging.properties